### PR TITLE
12 open camera

### PR DESCRIPTION
**Pull request by jacobstorms**
Verified that camera auto opens when ran on phone, removed debug features for clean slate view

### CODE REVIEW

File: AR-Poker-Hand-Identifier/ViewController.swift [hunk 1]
```diff
@@ -72,4 +53,6 @@ class ViewController: UIViewController, ARSCNViewDelegate {
         // Reset tracking and/or remove existing anchors if consistent tracking is required
         
     }
+
```
Comment (CollinDietz) [inline AR-Poker-Hand-Identifier/ViewController.swift:56]: EWS

File: AR-Poker-Hand-Identifier/Base.lproj/Main.storyboard [hunk 2]
```diff
@@ -1,23 +1,28 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="13122.16" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" useSafeAreas="YES" colorMatched="YES" initialViewController="BV1-FR-VrT">
+<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="14460.31" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" useSafeAreas="YES" colorMatched="YES" initialViewController="BV1-FR-VrT">
```
Comment (CollinDietz) [inline AR-Poker-Hand-Identifier/Base.lproj/Main.storyboard:2]: Why/ What is this change?

File: AR-Poker-Hand-Identifier/Base.lproj/Main.storyboard [hunk 3]
```diff
@@ -1,23 +1,28 @@
...
+    <device id="retina5_9" orientation="portrait">
+        <adaptation id="fullscreen"/>
+    </device>
     <dependencies>
-        <plugIn identifier="com.apple.InterfaceBuilder.IBCocoaTouchPlugin" version="13104.12"/>
+        <deployment identifier="iOS"/>
```
Comment (CollinDietz) [inline AR-Poker-Hand-Identifier/Base.lproj/Main.storyboard:7]: Why/ What is this change?

**Review by CollinDietz — CHANGES_REQUESTED** [review]
This PR seems to have a ton of changes, and the only thing I was imagining was removing the jet and commenting out the debug bar. Are they needed/ wanted? I can't tell what most of them do.

May be cleaner to just start from scratch and do just the handful of things

**Comment by jacobstorms** [comment]
I think the changes are artifacts from me playing around with the sceneview settings - they're all arbitrary for what we're doing, but I'll close this and create a new, cleaner one

**Pull request closed without merging**